examples/OpenGL45/arrayReductionByBitwiseOr_version2.shadertrap

```diff
@@ -0,0 +1,65 @@
+
+CREATE_BUFFER input_array_origin SIZE_BYTES 1024 INIT_VALUES uint 0 2 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 52 54 56 58 60 62 64 66 68 70 72 74 76 78 80 82 84 86 88 90 92 94 96 98 100 102 104 106 108 110 112 114 116 118 120 122 124 126 128 130 132 134 136 138 140 142 144 146 148 150 152 154 156 158 160 162 164 166 168 170 172 174 176 178 180 182 184 186 188 190 192 194 196 198 200 202 204 206 208 210 212 214 216 218 220 222 224 226 228 230 232 234 236 238 240 242 244 246 248 250 252 254 256 258 260 262 264 266 268 270 272 274 276 278 280 282 284 286 288 290 292 294 296 298 300 302 304 306 308 310 312 314 316 318 320 322 324 326 328 330 332 334 336 338 340 342 344 346 348 350 352 354 356 358 360 362 364 366 368 370 372 374 376 378 380 382 384 386 388 390 392 394 396 398 400 402 404 406 408 410 412 414 416 418 420 422 424 426 428 430 432 434 436 438 440 442 444 446 448 450 452 454 456 458 460 462 464 466 468 470 472 474 476 478 480 482 484 486 488 490 492 494 496 498 500 502 504 506 508 510
+
+CREATE_BUFFER partial_bitwise_or SIZE_BYTES 64 INIT_VALUES uint 0 62 126 126 254 254 254 254 510 510 510 510 510 510 510 510  
+CREATE_BUFFER partial_bitwise_or_expected SIZE_BYTES 64 INIT_VALUES uint 30 62 126 126 254 254 254 254 510 510 510 510 510 510 510 510 
+CREATE_BUFFER result SIZE_BYTES 4 INIT_VALUES uint 0
+CREATE_BUFFER expected SIZE_BYTES 4 INIT_VALUES uint 510
+
+BIND_SHADER_STORAGE_BUFFER BUFFER input_array BINDING 0
+BIND_SHADER_STORAGE_BUFFER BUFFER partial_bitwise_or BINDING 1
+BIND_SHADER_STORAGE_BUFFER BUFFER result BINDING 2
+
+DECLARE_SHADER shader KIND COMPUTE
+#version 450
+
+layout(local_size_x=1, local_size_y=1, local_size_z=1) in;
+
+layout(std430, binding = 0) buffer input_array {
+  uint[256] data1;
+};
+
+layout(std430, binding = 1) buffer partial_bitwise_or {
+  uint[16] data2;
+};
+
+layout(std430, binding = 2) buffer result {
+  uint res;
+};
+
+void main() {
+	for(uint i = gl_WorkGroupID[0]*16; i < gl_WorkGroupID[0]*16 + 16; ++i) {
+		data2[gl_WorkGroupID[0]] |= data1[i];
+	}
+	atomicOr(res, data2[gl_WorkGroupID[0]]);
```

Comment:
Again, please be consistent with indentation - worth checking carefully throughout the PR.